ecs_fargate/datadog_checks/ecs_fargate/data/conf.yaml.example

```diff
@@ -1,10 +1,10 @@
 init_config:
 
 instances:
-  ## @param timeout - integer - required
+  ## @param timeout - integer - optional - default: 5
   ## Time to wait on the Fargate metadata API before timeout in second
   #
-  - timeout: 5
```

Comment:
If you do this there's no default instance anymore 😬

Reply:
@mfpierre why do we need a configuration file to setup an integration if all parameters are optional?